docs/UserGuide.md

```diff
@@ -163,6 +170,8 @@ Format: `edit ENTRY_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [nk/NE
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* Editing the name of a person to be the exact same name as another person currently in the contact book
+    (case-sensitive) will cause the command to fail.
```

Comment:
Are we checking equality of person based on name? If we are then maybe this behaviour should be handled without case sensitivity?

Reply:
Changing the validity check counts as a feature change i think